### PR TITLE
Remove hardcoded alpha domain

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Remove instances of notification.alpha.canada.ca in this repository.

Note: it was not required to change the SVG file, just did it to avoid returning a match when searching for `notification.alpha.canada.ca` in the `app` folder

### CODE REVIEW

File: app/templates/views/service-settings/sending_domain.html [hunk 1]
```diff
@@ -23,13 +23,13 @@
     {{ textbox(
       form.sending_domain,
       width='1-1',
-      hint=_('Default notification.alpha.canada.ca'),
+      hint=_('Default {}').format(sending_domain),
```
Comment (AntoineAugusti) [inline app/templates/views/service-settings/sending_domain.html:26]: This variable was already passed to the template, just not used

**Comment by brdunfield** [comment]
Screen Shot 2020-11-23 at 1 28 23 PM
😂 
It's working!

**Review by brdunfield — APPROVED** [review]
🦑

**Comment by AntoineAugusti** [comment]
Yeah, the API endpoint will be shown locally as `https://api.localhost:6012` but hey 🤷

**Pull request merged**